scalafix/output/src/main/scala/rsc/tests/RscCompat.scala

```diff
@@ -0,0 +1,126 @@
+    val compoundType5: _root_.scala.AnyRef with _root_.rsc.tests.RscCompat_Test.TypesHelpers.A with _root_.rsc.tests.RscCompat_Test.TypesHelpers.B = new A with B
+    val compoundType6: _root_.scala.AnyRef with _root_.rsc.tests.RscCompat_Test.TypesHelpers.A with _root_.rsc.tests.RscCompat_Test.TypesHelpers.B { def k: _root_.scala.Int } = new A with B { def k: Int = ??? }
+    val compoundType7: _root_.rsc.tests.RscCompat_Test.TypesHelpers.A with (_root_.scala.collection.immutable.List[T] forSome { type T }) with _root_.rsc.tests.RscCompat_Test.TypesHelpers.B = ??? : A with (List[T] forSome { type T }) with B
+
+    val annType1: _root_.rsc.tests.RscCompat_Test.TypesHelpers.C @_root_.rsc.tests.RscCompat_Test.TypesHelpers.ann = ??? : C @ann
+
+    val existentialType1: _root_.scala.Any = ??? : T forSome { type T }
+    val existentialType2: _root_.scala.collection.immutable.List[_root_.scala.Any] = ??? : List[_]
+
+    val universalType1: _root_.rsc.tests.RscCompat_Test.TypesHelpers.H[({ type λ[U] = _root_.scala.collection.immutable.List[U] })#λ] = ??? : H[({ type L[U] = List[U] })#L]
+
+    val byNameType: (=> _root_.scala.Any) => _root_.scala.Any = ??? : ((=> Any) => Any)
+    // FIXME: https://github.com/twitter/rsc/issues/144
+    // val repeatedType = ??? : ((Any*) => Any)
+  }
+
+  class Bugs {
+    val Either: _root_.scala.util.Either.type = scala.util.Either
+
+    implicit def order[T]: _root_.java.lang.Object with _root_.scala.`package`.Ordering[_root_.scala.`package`.List[T]] = new Ordering[List[T]] {
+      def compare(a: List[T], b: List[T]): Int = ???
+    }
+
+    val gauges: _root_.scala.collection.immutable.List[_root_.scala.Int] = {
+      val local1 = 42
+      val local2 = 43
+      List(local1, local2)
+    }
+
+    val clazz: _root_.java.lang.Class[T1] forSome { type T1 } = Class.forName("foo.Bar")
+
+    val compound: _root_.scala.AnyRef { def m(x: _root_.scala.Int): _root_.scala.Int } = ??? : { def m(x: Int): Int }
+
+    val loaded: _root_.scala.collection.immutable.List[_root_.java.lang.Class[T1] forSome { type T1 }] = List[Class[_]]()
+
+    val ti: Types[Int] = ???
+    val innerClass1: _root_.rsc.tests.RscCompat_Test.Types[_root_.scala.Predef.String]#X = new Types[String]().x
+    val innerClass2: _root_.rsc.tests.RscCompat_Test.Types[_root_.scala.Int]#X = ??? : Types[Int]#X
+    val innerClass3: Bugs.this.ti.X = ti.x
+    val innerClass4: _root_.java.util.Base64.Decoder = Base64.getMimeDecoder
+
+    val param: _root_.scala.AnyRef { val default: _root_.scala.Boolean } = new { lazy val default = true }
+    val more1: _root_.scala.AnyRef { var x: _root_.scala.Int;  } = new { var x = 42 }
+    val more2: _root_.scala.AnyRef { def foo(implicit x: _root_.scala.Int, y: _root_.scala.Int): _root_.scala.Int } = new { def foo(implicit x: Int, y: Int) = 42 }
+    val more3: _root_.scala.AnyRef { def bar: _root_.scala.Int } = new { implicit def bar = 42 }
```

Comment:
Do we still use _root_ in semanticdb 4.0?

Reply:
We no longer use the `_root_` prefix in the recent versions of SemanticDB, however we need this prefix in RscCompat to avoid inadvertent name capture until https://github.com/twitter/rsc/issues/141 is fixed.